doc/source/workflows/basics.rst

```diff
@@ -244,15 +244,21 @@ The following error handling flags can be either set in the task decorator or vi
     workflow.run(handle_errors.bind(r2))
 
 
-- If ``max_retries`` is given, the task will be retried for the given number of times if an exception is raised. It will only retry for the application level error. For system errors, it's controlled by ray. By default, ``max_retries`` is set to be 3.
+- Ray Workflow also supports ``max_retries`` as in Ray tasks. If ``max_retries`` is given, the task will be retried for the given number of times if an exception is raised. It will only retry for the application level error. For system errors, it's controlled by ray. By default, ``max_retries`` is set to be 3.
```

Comment:
I thought we disable ray layer max_retries right?

Reply:
yes, but both the API and semantics are same as Ray. for the implementation, we disabled it; but for the behavior, it is like we are doing the exactly same stuff